spring-boot/admin/src/main/java/eu/coatrack/admin/service/OAuthUserDetailsService.java

```diff
@@ -65,19 +78,20 @@ private OAuth2User getLoggedInUser() {
     }
 
     public String getLoginNameFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("login");
+        return getLoggedInUser().getAttribute(oauthUserInfoAttributeNameForLogin);
     }
 
     public String getNameFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("name");
+        return getLoggedInUser().getAttribute(oauthUserInfoAttributeNameForName);
     }
 
+    // The toString was added, due to the fact that the organizations attribute in Keyrock is a List, while in Github is a String
     public String getCompanyFromLoggedInUser() {
-        return getLoggedInUser().getAttribute("company");
+        return getLoggedInUser().getAttribute(oauthUserInfoAttributeNameForCompany).toString();
```

Comment:
I just created a minimal example what the output would look like:
```
ArrayList<String> strings = new ArrayList<>();
strings.add("hello");
strings.add("world");
System.out.println(strings.toString());
```
toString() of a list generates an output like: `[hello, world]`. The squared brackets look odd and the user should not see this. I have two ideas:
* We just use the first company element here,
* or we generate a string without square brackets here.

Is the first company of the list somehow more important than the others? If not, the second approach would be better.

Reply:
That is a good point, the issue however is that now when CoatRack is deployed with KeyRock, the json retrieved in the userEndpoint of keyrock will always contain an empty list, so If I do a `.get(0)` in order to get the first element, wouldn't I get a nullPointException?

Reply:
This still needs to be better discussed, because for KeyRock the organizations are in a list, while for Github, you can only have one organization that is in a String